internal/consensus/wal_test.go

```diff
@@ -82,10 +84,37 @@ func TestWALTruncate(t *testing.T) {
 
 func TestWALEncoderDecoder(t *testing.T) {
```

Comment:
Probably a good test would be to include logic in the `func TestWALMsgProto(t *testing.T)` test to ensure the conversions to proto work fine especially in the edge cases where the `ReceiveTime` can be `nil`. 

Noticed that in this unit test there is no test for this particular WAL message

Reply:
Thanks, @andynog 
Yeah, I need to add a test that serializes in the old version and deserializes in the new one